src/cmd/bootstrap.ts

```diff
@@ -48,6 +54,44 @@ const generateLooseSchema = () => {
   }
 }
 
+const hfValuesOrEmpty = async ({ skipCache }: { skipCache?: boolean }): Promise<Record<string, any>> => {
```

Comment:
Returning undefined would require aditional logic on checking it the returend value type and acting upon that result.
Returning empty dict allows to avoid that logic.
I revereted named parameter.

Reply:
Yes I understand, but that is not separating concerns and leads to assumptions...